### ISSUE TITLE
polygonscan verification error

### ISSUE THREAD

**Issue by stringcheeseslut** [HashLips/solidity_smart_contracts#10]
trying to figure out how to resolve this issue. much appreciated in advance.

Compiler debug log:
 Error! Unable to generate Contract ByteCode and ABI (General Exception, unable to get compiled [bytecode])
 For troubleshooting, you can try compiling your source code with the Remix - Solidity IDE and check for exceptions

ParserError: Source "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol" not found: File import callback not supported
 --> myc:7:1:
  |
7 | import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol"
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

ParserError: Source "@openzeppelin/contracts/access/Ownable.sol" not found: File import callback not supported
 --> myc:8:1:
  |
8 | import "@openzeppelin/contracts/access/Ownable.sol"
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by sultan-710** [comment]
Yes i have the same issue when i wanna verify my contract address and i get the exact same error. plz anyone help :)

**Comment by alessandrohk** [comment]
same issue here, what should i do?
